Website/src/platform/FieldSerializer/DemoImageFieldSerializer.cs

```diff
@@ -30,7 +31,55 @@ protected override void WriteRenderedValue(Field field, JsonTextWriter writer)
         protected virtual string GetRenderedValue(Field field, SerializationOptions options = null)
         {
             if (string.IsNullOrWhiteSpace(this._renderedValue))
+            {
                 this._renderedValue = this.RenderField(field, options != null && options.DisableEditing).ToString();
+            }
+
+            string transformation = "web";
+            Item item = field.Item;
+            string transformationFieldName = field.Name + "Transformation";
+            if (item?.Fields[transformationFieldName] != null)
+            {
+                transformation = item.Fields[transformationFieldName].Value;
+            }
+
+            var fieldValue = this._renderedValue;
+
+            if (fieldValue.Contains("stylelabs-content-id"))
```

Comment:
This entire block of code is a rather resource-intensive operation that will run hundreds of times. Can we instead hard-code "t=web" in all serialized image fields?

Reply:
we definitely can. web is only 1500 width so its not too small or too big. let's discuss...

Reply:
After speaking with Topaz about this we have decided to slightly alter the code. It will first look if the field is an "Image" field before running the main block of code. This processor will still need to be run for each field but we think it is necessary due to the following:

- The main block of code will not run for most fields (ex: all system fields) but will still check all images including CH images added by the user that are not part of our original serialized content.
- It will likely be too error prone to force developers to update serialized YML files manually (adding the querystring value) whenever a new image is added to the demo.

We also decided on this approach because future

Reply:
this is fixed. Image field check added so only Image type fields goes thru this custom serializer.